src/components/add-todo.jsx

```diff
@@ -21,11 +21,69 @@ class AddTodo extends React.Component {
     }
   }
 
+  componentDidMount() {
+    this.textInput.focus();
+  }
+
   render () {
     return (
-      <form onSubmit={this.handleSubmit.bind(this)}>
-        <input type="text" value={this.state.value} onChange={this.handleChange.bind(this)}/>
-        <button type="submit">Add todo</button>
+      <form onSubmit={this.handleSubmit.bind(this)} className="add-todo">
+        <input
+          type="text"
+          value={this.state.value}
+          placeholder="New todo..."
+          ref={(input) => { this.textInput = input; }}
+          onChange={this.handleChange.bind(this)}
+          className="add-todo__input" />
+        <button className="add-todo__button">Add Todo</button>
```

Comment:
`add-todo__button` - is this some kind of a naming convention we are trying to follow here? If the class is called only `button` or `add-button` isn't it going to work? `styled-jsx` should be scoping the styles properly.

Reply:
yes,  smth like `block__element--state` like here: https://en.bem.info/methodology/naming-convention/#alternative-naming-schemes . Yes, `button`  is going to work here, that is more for global styling convention. But naming it this way you can use selectors when styling so you can DRY a little bit - like here: https://github.com/atokan/AtokanTodo/pull/1/files#diff-c49cdcd8cec9d87172315c662e020625R30

Reply:
Sounds good. Let's try to use this convention